### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by CyanDevs**
Signed-off-by: Chris Yan <chrisyan@microsoft.com>

### CODE REVIEW

File: config/jenkins/configuration/clouds.yml [hunk 1]
```diff
@@ -250,7 +250,7 @@ jenkins:
       cloudName: "AzureVMAgentsEastUs"
       configurationStatus: "pass"
       deploymentTimeout: 1200
-      newResourceGroupName: "<AZURE_VM_RESOURCE_GROUP>-westus"
```
Comment (BRMcLaren) [inline config/jenkins/configuration/clouds.yml:253]: might want to postfix these with 2, as that is the region bein used to avoid.. the future problem

Comment (CyanDevs) [reply config/jenkins/configuration/clouds.yml:253]: What do you mean? Are you talking about West US 2?

Comment (BRMcLaren) [reply config/jenkins/configuration/clouds.yml:253]: Yes, i am talking about west us 2, I believe that is the Azure deployment region being used.

Comment (CyanDevs) [reply config/jenkins/configuration/clouds.yml:253]: Changed below

**Pull request merged**